$Id$
$MirOS: ports/www/lynx/patches/patch-makefile_in,v 1.2 2007/05/20 12:06:19 tg Exp $
--- lynx2-8-7.orig/makefile.in	2007-05-18 00:53:22.000000000 +0200
+++ lynx2-8-7/makefile.in	2009-06-13 11:46:29.000000000 +0200
@@ -161,13 +161,13 @@ SITE_DEFS = # Your defines here
 # for installation of local execution links, please see the file userdefs.h
 
 WWW_CFLAGS = \
-	LY_CFLAGS="$(CFLAGS)" \
-	CPPFLAGS="$(CPPFLAGS)" \
+	LY_CFLAGS='${CFLAGS}' \
+	CPPFLAGS='${CPPFLAGS}' \
 	LYFLAGS="$(SITE_LYDEFS)"
 
 SRC_CFLAGS = \
-	CFLAGS="$(CFLAGS)" \
-	CPPFLAGS="$(CPPFLAGS)" \
+	CFLAGS='${CFLAGS}' \
+	CPPFLAGS='${CPPFLAGS}' \
 	LIBS="$(LIBS) $(RESOLVLIB) $(WAISLIB) $(SITE_LIBS)" \
 	SITE_DEFS="$(SITE_DEFS)" \
 	WWWINC=$(WWWINC) \
